container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/SearchCluster.java

```diff
@@ -76,6 +96,9 @@ public SearchCluster(String clusterId, DispatchConfig dispatchConfig, int contai
         for (Node node : nodes)
             nodesByHostBuilder.put(node.hostname(), node);
         this.nodesByHost = nodesByHostBuilder.build();
+        hitEstimator = ((0.0 < dispatchConfig.topKProbability()) && (dispatchConfig.topKProbability() < 1.0))
```

Comment:
Would be better to move this into the estimator I think. (And then also move it to the top level as a package-visible class).

Reply:
Agree, done.